Analysis/ScanLibraries/include/XiaData.hpp

```diff
@@ -89,7 +89,9 @@ class XiaData {
     ///@return True if this channel was generated on the module
     bool IsVirtualChannel() const { return isVirtualChannel_; }
 
-    ///@return The baseline as it was calculated on the module
+    ///@return The baseline as it was calculated on the module.
```

Comment:
> ///The calculation was done using a trapezoidal filter.
    ///This requires that ESUMS parameter be enabled on the modules during acquisition.
    ///The baseline returned by this function can not be used with values calculated from the trace, like the QDC.
    ///It also can not be subtracted from the trace like we do with the calculated baseline (HighResTimingData::GetAveBaseline())
    ///@return The baseline as it was calculated on the module, in trapezoidal filter units.
    double GetBaseline() const { return baseline_; }

I would prefer
```c++
///@return The baseline as calculated on-board the Pixie-16 modules using the 
/// energy filter. This parameter is only set if the data set contains the Energy 
/// Sums in the list mode data. This baseline cannot be used in conjunction with 
///trace information. 
```

Also, make sure you always conform to the 80 character line limits. See the [Style Guidelines](https://github.com/pixie16/paass/wiki/Style-Guidelines#formatting)